### DISCUSSION TITLE
N64 blueretro adapter from 8bitmods issues

### DISCUSSION THREAD

**Discussion by Insaneo99** [darthcloud/BlueRetro#570] [General]
One of my adapters purchased from 8bitmods for the N64 using blueretro has two issues. i was asked to confirm on here if its a hardware issue before they would replace it.

im using it with a nintendo switch online 64 controller. Only 1 adapter plugged in. On whatever firmware it shipped with.

issue 1: it unpairs sometimes and wont repair unless i hold the button on it for 10 seconds to reset the adapter to factory. Happened 3-4 times so far. Happens when 64 is still powered on and controller was being used as well as booting it up next time.

issue 2: random spikes in input lag and theres nothing blocking or interfering with line of sight or barriers between controller and adapter. Happens once maybe every couple hours of game time.

any help would be appreciated thanks.

**Comment by darthcloud** [comment]
I means if you got two identical dongle and one work 100% and the other doesn't it got to be a HW issues.

Sound like antenna is bad on that one.

**Reply by Insaneo99** [reply]
I cant seem to update either adapters firmware. It gets to a random %. 39, 54, 83% then stops and disconnects before 100%. Tried multiple times using a laptop and chrome with hw1 firmware and the n64.bin

**Reply by darthcloud** [reply]
The web work fine on some laptop and doesn't on other. Try to borrow another laptop.
